src/api/notes/dto-methods.ts

```diff
@@ -7,11 +7,25 @@
 import { NoteDto } from './types'
 import { NoteDetails } from '../../redux/note-details/types'
 import { DateTime } from 'luxon'
-import { initialState } from '../../redux/note-details/reducers'
+import { initialState } from '../../redux/note-details/initial-state'
 
+/**
+ * Converts a note DTO from the API to a {@link NoteDetails} object.
+ * Note that the documentContent will be set but the markdownContent and rawFrontmatterContent are empty.
+ * Therefore, after getting a note from the API, you need to call the {@link setNoteContent} method.
+ * @param note The NoteDTO as defined in the backend.
+ * @return The NoteDetails object corresponding to the DTO.
+ */
 export const noteDtoToNoteDetails = (note: NoteDto): NoteDetails => {
```

Comment:
If I see it correctly then this method is only used in the reducer. And only to overwrite the state with the note from the DTO. Therefore this method should be moved to the reducer.